core/expressions.py

```diff
@@ -525,6 +543,13 @@ class Operator(BinaryOperation.Operator):
         In = 9
         NotIn = 10
 
+        def reverse_operator(self):
+            """Returns the reverse operator of this operator."""
+            try:
+                return BinaryComparisonOperation.Operator.REVERSE_OPERATOR[self]
```

Comment:
I don't see any warnings for unresolved attribute and the code is correct (covered by tests).
the line
`Operator.REVERSE_OPERATOR=...`
adds that attribute to the Operator inner class; this construction is necessary because otherwise the `REVERSE_OPERATOR` field would get interpreted as an Enum constant